package.json

```diff
@@ -13,8 +13,8 @@
     "test:ci": "jest --passWithNoTests --maxWorkers 4",
     "typecheck": "tsc --noEmit",
     "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
-    "lint:fix": "yarn lint --fix",
-    "server": "docker compose -f ./devenv/docker-compose.yaml up --build -d",
+    "lint:fix": "yarn run lint --fix && prettier --write --list-different .",
+    "server": "docker compose up --build",
```

Comment:
Do we want to change it? 🤔

Reply:
Possibly got overridden because of https://github.com/grafana/plugin-tools/issues/820 ([Slack](https://raintank-corp.slack.com/archives/C01C4K8DETW/p1754665848310939))